docs/EVMC.md

```diff
@@ -40,6 +40,7 @@ can be referenced as EVMC ABIv3 or just EVMC 3.
 2. **Host** – An entity controlling the VM. 
    The Host requests code execution and responses to VM queries by callback 
    functions. This usually represents an Ethereum Client.
+3. **null hash** – a hash which all bytes are zeros.
```

Comment:
"for which", "in which" maybe? Or maybe just call it "null bytes" for `get_block_hash`, too